lib/player.js

```diff
@@ -2791,6 +2791,7 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     this.playRateController_.set(rate);
 
     if (this.loadMode_ == shaka.Player.LoadMode.MEDIA_SOURCE) {
+      this.abrManager_.playbackRateChanged(rate);
```

Comment:
Since you added a method to a plugin interface, you'll need to account for backward compatibility.  I see two good options:

1. Check for playbackRateChange each time before calling it.
2. When this.abrManager_ is set, check for playbackRateChange at that time and attach an empty function if it's missing.

I would strongly prefer #2, because you can then use `shaka.Deprecate.deprecateFeature` at the same time to tell the app developer that they need to upgrade their plugin by v2.7.

Reply:
I add a check for playbackRateChanged and I followed your second option. Thanks!